frontend/src/lib/MetricsManager.test.ts

```diff
@@ -96,8 +100,6 @@ test("enqueues events before initialization", () => {
   expect(mm.identify.mock.calls[0][0]).toBe(SessionInfo.current.installationId)
   expect(mm.identify.mock.calls[0][1]).toMatchObject({
     authoremail: SessionInfo.current.authorEmail,
-    machineIdV1: SessionInfo.current.installationIdV1,
-    machineIdV2: SessionInfo.current.installationIdV2,
```

Comment:
It looks like `machineIdV1` and `machineIdV2` should still belong in this object call? It doesn't look like there's any conditional statements that would now remove these? Unless you're just covering these with the additional tests down below?

Reply:
Yea just moved the check for these into the tests below to make it a bit cleaner.